bing-dict.el

```diff
@@ -133,8 +133,52 @@ The value could be `synonym', `antonym', `both', or nil.")
                                          'face
                                          'font-lock-builtin-face))
 
+(defvar bing-dict--org-file "~/.orgs/vocabulary.org"
+  "The file where store the vocabulary.")
+
+(defvar bing-dict--org-file-title "Vocabulary"
+  "The title of the vocabulary org file.")
+
+(defvar bing-dict--save-search-result nil
+  "save bing dict search result or not.")
+
+
+(defun bing-dict--tidy-headlines ()
+  "remove extra spaces between stars and the headline text"
+  (save-excursion
+    (goto-char (point-min))
+    (while (re-search-forward "^\\*+\\([[:space:]][[:space:]]+\\)" (point-max) t)
```

Comment:
No, it seems can't, `org-heading-regexp` matches two groups -- stars are put in group 1 and the trimmed body in group 2. But in this case,
 
>"remove extra spaces between stars and the headline text" 

should only match the "space" after the stars.

This function is aim to make the vocabularies more good-looking in org mode.